packages/engine/Cargo.toml

```diff
@@ -43,6 +43,8 @@ tracing-appender = "0.2.0"
 tracing-error = "0.2.0"
 tracing-subscriber = { version = "0.3.7", features = ["env-filter", "json"] }
 tracing-texray = { version = "0.1.2", optional = true }
+# Only temporary, use the crates.io release when ArrayBuffer::new_backing_store_from_ptr is available
+v8 = { git = "https://github.com/hashdeps/rusty_v8-1", branch = "backing-store-from-ptr" }
```

Comment:
I've tried this locally and my build has taken over 20 minutes, it's talking about download git submodules to do with libcxx and such. As you've just suggested on Slack this may be due to us needing to use a git dependency here rather than a published crate.

Can you add something in the README saying how initial repo setup times may be long at the moment due to needing to maintain a fork of rusty_v8 but we expect them to improve significantly once PR #XXX is merged.

Reply:
It's not so much when the PR will be merged, we'll still need a fork then. It's when there is a new release after the PR is merged.
They seem to release a few times a month so it shouldn't take long once the PR is merged.

Reply:
Addressed in https://github.com/hashintel/hash/pull/414/commits/52ad8805e13867291b0e366ef128234d71297a5c